provision/acc_provision/acc_provision.py

```diff
@@ -538,6 +541,23 @@ def config_adjust(args, config, prov_apic, no_random):
         adj_config["aci_config"]["vmm_domain"]["injected_cluster_type"] = ""
     if not config["aci_config"]["vmm_domain"].get("injected_cluster_provider"):
         adj_config["aci_config"]["vmm_domain"]["injected_cluster_provider"] = ""
+
+    if config["sriov_config"].get("enable") is True:
+        if config["sriov_config"]["device_info"].get("devices") != None :
```

Comment:
if 'devices' in config["sriov_config"]["device_info"]:

Reply:
resolved